### DISCUSSION TITLE
FSOD问题

### DISCUSSION THREAD

**Discussion by Bitdopad** [Zhengzkang/Few-shot-object-detection#1] [General]
可以参考我今天发的code，里面有可视化代码

**Comment by Zhengzkang** [comment]
有什么问题？

**Reply by Bitdopad** [reply]
你好，我想问一下训练好FCT双支模型得到pth文件后，对于给定的一张场景图片和一张目标图片，怎么输入呢？outputs=modle(inputs)里的inputs是什么呢？
